### PR TITLE
Cube cone arm positions

### LINKED ISSUE #123: Add code to know if robot is set to cone vs cube mode
We want to adjust the top/middle heights based on if we have a cone vs cube.  Currently, when the cone/cube buttons are set, only the LEDs are set.  The gamePiece state needs to be remembered so that the ArmToTop and ArmToMiddle commands can query the mode.

### LINKED ISSUE #124: Update ArmToTop/ArmToMiddle to set position depending on cone vs cube.

### PR DESCRIPTION
**Pull request by BrownGenius**
Resolves #123
Resolves #124

### CODE REVIEW

File: src/main/java/bhs/devilbotz/RobotContainer.java [hunk 1]
```diff
@@ -129,15 +143,25 @@ private void configureBindings() {
           .whileTrue(new ArmUp(arm, gripper))
           .onFalse(new ArmStop(arm));
 
-      new JoystickButton(leftJoystick, 6).onTrue(new SetLEDMode(arduino, LEDModes.SET_CONE));
+      new JoystickButton(leftJoystick, 6)
+          .onTrue(
+              new InstantCommand(
+                  () -> {
+                    // Reset odometry for the first path you run during auto
```
Comment (ParkerMeyers) [inline src/main/java/bhs/devilbotz/RobotContainer.java:150]: fix comment

File: src/main/java/bhs/devilbotz/RobotContainer.java [hunk 2]
```diff
@@ -129,15 +143,25 @@ private void configureBindings() {
...
+                    this.setGamePieceType(GamePieceTypes.CONE);
+                  }));
 
-      new JoystickButton(leftJoystick, 7).onTrue(new SetLEDMode(arduino, LEDModes.SET_CUBE));
+      new JoystickButton(leftJoystick, 7)
+          .onTrue(
+              new InstantCommand(
+                  () -> {
+                    // Reset odometry for the first path you run during auto
```
Comment (ParkerMeyers) [inline src/main/java/bhs/devilbotz/RobotContainer.java:158]: fix comment

**Review by ParkerMeyers — APPROVED** [review]
Looks good! We can test this at WPI today

**Comment by BrownGenius** [comment]
Included in #130

**Pull request closed without merging**